### PR TITLE
Unit Test for Magento\Sitemap\Model\Config\Backend\Priority

### PR DESCRIPTION
**Pull request by karyna-t**
### Description (*)
Covers Magento\Sitemap\Model\Config\Backend\Priority class with a unit test.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Sitemap/Test/Unit/Model/Config/Backend/PriorityTest.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            ->disableOriginalConstructor()
+            ->setMethods(['getValue'])
+            ->getMock();
+    }
+
+    /**
+     * @param string $value
+     * @dataProvider dataProviderTestBeforeSaveValueCorrect
+     */
+    public function testBeforeSave($value)
```
Comment (lbajsarowicz) [inline app/code/Magento/Sitemap/Test/Unit/Model/Config/Backend/PriorityTest.php]: "testBeforeSaveIsChainable" to explain what actually assertion do

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Just a little change to test name

File: app/code/Magento/Sitemap/Test/Unit/Model/Config/Backend/PriorityTest.php (hunk 1 above)
Comment (karyna-t) [reply app/code/Magento/Sitemap/Test/Unit/Model/Config/Backend/PriorityTest.php]: thanks, done

**Review by lbajsarowicz — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6629](https://jira.corp.magento.com/browse/ENGCOM-6629) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Delta** [comment]
**Notice:** QA not applicable

**Review by lbajsarowicz — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6629](https://jira.corp.magento.com/browse/ENGCOM-6629) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Golf** [comment]
failed Functional Tests B2B Not related to the changes in this PR

**Pull request merged**